src/app/child-dev-project/attendance/attendance.service.spec.ts

```diff
@@ -85,7 +118,9 @@ describe("AttendanceService", () => {
         "day"
       )
     ).toBeTrue();
-    expect(actualAttendances[0].events).toEqual([e1_1, e1_2]);
+    expect(cleanLoadedData(actualAttendances[0].events)).toEqual(
+      jasmine.arrayWithExactContents([e1_1, e1_2])
```

Comment:
In later tests `jasmine.arrayWithExactContents` is not used. Why is it necessary here?

Reply:
will be refactored in the next PR